From 37494bedb64dda4fb6621aa8a75e012d78ef4488 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=EB=A7=88=EB=88=84=EC=97=98?= <nalla@hamal.uberspace.de>
Date: Wed, 20 Jan 2016 07:22:10 +0100
Subject: [PATCH] digest-md5: Remove annoying debug message
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Every time an svn call is made, the message "DIGEST MD5 common mech free"
appears in the application event log viewer. Other communities (See
https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1372451.html)
are supressing the message with a logcheck snippet. Since we do not have
such a luxury in the msys2 environment, lets just remove the debug message
for good.

Signed-off-by: 마누엘 <nalla@hamal.uberspace.de>
---
 plugins/digestmd5.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/plugins/digestmd5.c b/plugins/digestmd5.c
index 4ab90dd..52ac085 100644
--- a/plugins/digestmd5.c
+++ b/plugins/digestmd5.c
@@ -1652,9 +1652,6 @@ static void digestmd5_common_mech_free(void *glob_context,
 	(digest_glob_context_t *) glob_context;
     reauth_cache_t *reauth_cache = my_glob_context->reauth;
     size_t n;
-
-    utils->log(utils->conn, SASL_LOG_DEBUG,
-	       "DIGEST-MD5 common mech free");
  
     /* Prevent anybody else from freeing this as well */
     my_glob_context->reauth = NULL;
-- 
2.7.0

